tests/testthat/test-brms.R

```diff
@@ -74,7 +74,7 @@ if (require("brms") && require("testthat") && require("insight") && require("htt
 
     out <- describe_posterior(model, effects = "all", components = "all", centrality = "mean", test = NULL)
     s <- summary(model)
-    expect_equal(s$fixed[, 1, drop = TRUE], out$Mean, check.attributes = FALSE, tolerance = 1e-3)
-    expect_equal(s$fixed[, 5, drop = TRUE], out$Rhat, check.attributes = FALSE, tolerance = 1e-1)
```

Comment:
`as.vector()` works here, in other situations, where we have no vector, the new argument is `ignore_attr = TRUE`.